src/ol_infrastructure/infrastructure/consul/__main__.py

```diff
@@ -175,16 +179,13 @@
     if subnet_object.availability_zone == "us-east-1e":
         continue
     instance_name = f"consul-{environment_name}-{count}"
-    salt_minion = OLSaltStackMinion(
-        f"saltstack-minion-{instance_name}",
-        OLSaltStackMinionInputs(minion_id=instance_name),
-    )
 
+    # Will cloud init be needed for pyinfra, rather than salt, setup?
```

Comment:
We will still be using cloud-init to populate the datacenter information and the appropriate auto-join string. You can see an example of that [here](https://github.com/mitodl/ol-infrastructure/blob/vector/src/ol_infrastructure/applications/edxapp/__main__.py#L722-L745)